src/__tests__/ariaAttributes.js

```diff
@@ -167,3 +167,32 @@ test('`pressed: true|false` matches `pressed` elements with proper role', () =>
   expect(getByRole('button', {pressed: true})).toBeInTheDocument()
   expect(getByRole('button', {pressed: false})).toBeInTheDocument()
 })
+
+test('`level` matches elements with `heading` role', () => {
+  const {getByRole, getAllByRole, queryByRole} = renderIntoDocument(
+    `<div>
+      <h1 id="heading-one">H1</h1>
+      <h2 id="first-heading-two">First H2</h2>
+      <h3 id="heading-three">H3</h3>
+      <div role="heading" aria-level="2" id="second-heading-two">Second H2</div>
+    </div>`,
+  )
+
+  expect(getByRole('heading', {level: 1})).toBeInTheDocument()
```

Comment:
Can we make these explicit so that it's clear that we didn't accidentally return the h2? We are later explicit for the level 2 heading so either assert everywhere that any element was returned or everywhere which exact element was returned.